doc/integrations/duckdb.md

```diff
@@ -1,4 +1,6 @@
 ---
+og:image: https://jupysql--685.org.readthedocs.build/en/685/_static/duckdb.png
```

Comment:
is the conclusion that relative paths do not work?

Reply:
I wouldn't say it takes relative path since, what it does is ogp_site_url + og:image, in the end the image is referenced from link: https://jupysql--685.org.readthedocs.build/en/685/_static/duckdb.png

Reply:
The former one works, I have pushed the code